addons/mass_mailing_crm/models/utm.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class UtmCampaign(models.Model):
+    _inherit = 'utm.campaign'
+
+    ab_testing_winner_selection = fields.Selection(selection_add=[('crm_lead_count', 'Leads')])
```

Comment:
This field has ``group_sale_salesman``, meaning it could be not achievable ... purpose was to hide stat button for non sales people (would crash when trying to see leads). Using it in a computation is probably error prone ...